prosemirror-lwdita/src/commands.ts

```diff
@@ -561,16 +557,15 @@ export function getTree(pos: ResolvedPos, depth = 0) {
  *
  * @param state - The EditorState object
  * @param dispatch - A function to be used to dispatch a transaction
- * @param view - The EditorView object
  * @returns Boolean
  */
-export function enterPressed(state: EditorState, dispatch?: (tr: Transaction) => void, view?: EditorView) {
-  let { $from, empty } = state.selection;
+export function enterPressed(state: EditorState, dispatch?: (tr: Transaction) => void) {
+  let { $from } = state.selection;
```

Comment:
Why does one of these have a `$` prefix?

Reply:
corresponding to https://prosemirror.net/docs/ref/#state.Selection.$from.